helper-cli/src/main/kotlin/commands/packagecuration/CreatePackageCurationCommand.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.packagecuration
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.UsageError
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import java.io.IOException
+
+import org.ossreviewtoolkit.helper.common.getSplitCurationFile
+import org.ossreviewtoolkit.model.FileFormat
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.model.PackageCurationData
+import org.ossreviewtoolkit.model.writeValue
+import org.ossreviewtoolkit.utils.expandTilde
+
+internal class CreatePackageCurationCommand : CliktCommand(
+    help = "Create a single curation file for a package id complying to the directory structure created by the" +
+            "`split-curations` sub-command."
```

Comment:
How about not mentioning the other command, because that hierarchical structure will be
common anyway I presume.

Create a curation file for a package id using a hierarchical directory structure.

Reply:
Changed it to your suggestion. Do you think it makes sense to mention the structure from `split-curations` again?
https://github.com/oss-review-toolkit/ort/blob/b093d2ec08b69abfe73f4eacdb8564f58479edd1/helper-cli/src/main/kotlin/commands/SplitCurations.kt#L37

Reply:
Thanks.